spec/calendar.html

```diff
@@ -1126,9 +1116,9 @@ <h1>Temporal.Calendar.prototype.mergeFields ( _fields_, _additionalFields_ )</h1
       <emu-alg>
         1. Let _calendar_ be the *this* value.
         1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
-        1. Assert: _calendar_.[[Identifier]] is *"iso8601"*.
         1. Set _fields_ to ? ToObject(_fields_).
         1. Set _additionalFields_ to ? ToObject(_additionalFields_).
+        1. Assert: _calendar_.[[Identifier]] is *"iso8601"*.
```

Comment:
why move this assertion? shouldn't it still be true in its original position?

Reply:
It's for alignment with the steps of [ECMA-402 Temporal.Calendar.prototype.mergeFields](https://tc39.es/proposal-temporal/#sup-temporal.calendar.prototype.mergefields): 870147bf5b42192f555aa21c59cafdbf2471dfd7